docs/data/joy/getting-started/overview/overview.md

```diff
@@ -22,44 +18,48 @@ We're adding new components and features regularly, and you're welcome to contri
 
-Try Joy UI if you appreciate the comprehensiveness and reliability of Material UI, but don't need all of the additional baggage that comes along with Material Design.
+### Highly customizable
 
-## Principles
+You should feel inspired and empowered to change, extend, and revamp Joy UI's appearance and behavior with ease.
+Drawing from many years of experience with Material UI, Joy UI applies new approaches to customization, enabling you to customize every piece of the components to match your unique design.
 
-### Include only what's essential
+Using [Gatsby's documentation side nav](https://www.gatsbyjs.com/docs/) as an example, the demo below shows how to customize the [List](/joy-ui/react-list/) component to look like it using built-in CSS variables.
 
-Joy UI should work with the least amount of effort possible.
-We're striving for the essential only, both in the component API and design (look and feel).
+{{"demo": "../../components/list/ExampleCollapsibleList.js"}}
 
-Components should have only what they need to do the job.
-What is considered essential is drawn from MUI's experience over the years developing component libraries, as well as from benchmarks of modern API and design guidelines—especially when it comes to developing web apps.
+### Developer experience
 
-### Make it beautiful out of the box
+One of the main goals of Joy UI is to spark joy in the creative process of building apps.
+Promoting an unrivaled developer experience is at the heart of this.
 
-Joy UI needs to be simple yet characteristic.
-Visual attributes such as scale, size, and density should be consistent across all of the components so they live together nicely.
+Take as an example of this the automatic adjustment of the [Input](/joy-ui/react-input/) component, whereby by customizing its border radius, all components inside it will adapt accordingly, ensuring a polished design _and_ saving you time.
 
-We aim to spark delight with simplicity and attention to detail.
-You should feel like your UI looks great from the start.
+{{"demo": "../../main-features/automatic-adjustment/InputVariables.js"}}
 
-### Encourage creativity
+### Accessibility
 
-Working with Joy UI shouldn't feel stiff or rigid—you should feel inspired and empowered to change, extend, and revamp Joy UI's appearance and behavior with ease.
-We want it to become your go-to starting point for building UIs.
+Joy UI components are built on top of [Base UI's unstyled components and low-level hooks](/base-ui/getting-started/), giving you many accessibility features support out of the box.
+We do our best to make all components screen reader-friendly and offer suggestions for optimizing accessibility throughout our documentation.
 
-### Prioritize developer experience
+One example of this is the [Input](/joy-ui/react-input/) component that needs to have a descriptive label linked to help users understand its purpose.
```

Comment:
Since this is about the general concept of an input.

```suggestion
For example, an input component needs to have a descriptive label to help users understand its purpose.
```